cli/app.go

```diff
@@ -610,6 +610,31 @@ viam board upload --name=orin --org="my org" --version=1.0.0 file.json`,
 					},
 					Action: UploadBoardDefsAction,
 				},
+				{
+					Name:  "get",
+					Usage: "get a board definition package",
+					Description: `download a json board definitions file for generic linux boards.
+Example:`,
+					UsageText: "viam board download <name> <organization> <version> [other options]",
+					Flags: []cli.Flag{
+						&cli.StringFlag{
+							Name:     boardFlagName,
+							Usage:    "name of thr board definitions file to download",
+							Required: true,
+						},
+						&cli.StringFlag{
+							Name:     organizationFlag,
+							Usage:    "Organization that hosts the board defintions file",
```

Comment:
```suggestion
							Usage:    "organization that hosts the board definitions file",
```
(I think this typo also slipped into upload)